### ISSUE TITLE
Unmaintained repository?

### ISSUE THREAD

**Issue by onlyjob** [stylish-userstyles/stylish#329]
What happened to this repository?  
No commits since 2016; no tag for [latest release](https://addons.mozilla.org/en-US/firefox/addon/stylish/versions/)?

Is this repository abandoned?

**Comment by satyapramodh** [comment]
i contacted them via the site's contact form so lets see. looks like the chrome and firefox extensions in their browser store have an advanced version than thats here. so thats weird

**Comment by satyapramodh** [comment]
@JasonBarnabe do you know if this is being maintained?

**Comment by daniel-barrows** [comment]
Stylish is maintained (see recent versions of the [Firefox Addon](https://addons.mozilla.org/en-US/firefox/addon/stylish/versions/)), but doesn't share a public source code repository. Indeed, this repository has been abandoned. You can access the source code by downloading the Firefox XPI addon and [unpack it as a zip archive](https://www.lifewire.com/xpi-file-2622582). For another discussion on this topic, see https://forum.userstyles.org/discussion/comment/117061/

**Comment by onlyjob** [comment]
What a shame... :( These days maintaining an open source project without public VCS (like Oracle does) is an embarrassment... Stylish deserves a public repository...

I suppose it might mean the end of Stylish in Debian. I'll see if [Stylus](https://github.com/openstyles/stylus) is a decent successor...

**Comment by satyapramodh** [comment]
Yes. it is sad. Thanks @onlyjob for letting me know about stylus. i will see if i can help that repo now.

**Comment by onlyjob** [comment]
I've learned about Stylus from https://forum.userstyles.org/discussion/comment/117061/ so we should thank @daniel-barrows. :)
